### PR TITLE
DAT-10810 QC: checks show changes (--show-cols flag, drop some, etc)

### PR DESCRIPTION
**Pull request by dyadyushko**
## Impact

- [X] Bug fix (non-breaking change which fixes expected existing functionality)
- [X] Enhancement/New feature (adds functionality without impacting existing logic)
- [X] Breaking change (fix or feature that would cause existing functionality to change)
 
## Description

Minor changes within TableOutput class.
Adding toLowerWithoutWhitespaces method to StringUtil class.
example: 
    "  InPu t " -> "input"

## Additional Context
Changes were made to support https://github.com/liquibase/liquibase-pro/pull/628.

### CODE REVIEW

**Review by wwillard7800 — APPROVED** [review]

File: liquibase-core/src/main/java/liquibase/util/StringUtil.java [hunk 1]
```diff
@@ -1031,6 +1031,14 @@ public static byte[] getBytesWithEncoding(String string) {
         return string.getBytes(StandardCharsets.UTF_8);
     }
 
+    /**
+     * @param value string to process
+     * @return string without any whitespaces formatted to lowercase.
+     */
+    public static String toLowerWithoutWhitespaces(String value) {
+        return value.toLowerCase().replaceAll("\\s+", "");
+    }
```
Comment (StevenMassaro) [inline liquibase-core/src/main/java/liquibase/util/StringUtil.java:1040]: Should probably handle the null case here. Maybe something like:

```suggestion
    public static String toLowerWithoutWhitespaces(String value) {
        if (value == null) {
            return null;
        }
        return value.toLowerCase().replaceAll("\\s+", "");
    }
```

File: liquibase-core/src/test/groovy/liquibase/util/StringUtilTest.groovy [hunk 2]
```diff
@@ -118,6 +119,17 @@ class StringUtilTest extends Specification {
...
+    def "to lower without whitespaces"() {
+        expect:
+        StringUtil.toLowerWithoutWhitespaces(string) == expected
+
+        where:
+        string                | expected
+        "First Value"         | "firstvalue"
+        "secondValue"         | "secondvalue"
+        "   Third Value   \n" | "thirdvalue"
+    }
```
Comment (StevenMassaro) [inline liquibase-core/src/test/groovy/liquibase/util/StringUtilTest.groovy:131]: Add a test for the null case here. Maybe also add a test for empty string.

File: liquibase-core/src/main/java/liquibase/util/StringUtil.java [hunk 3]
```diff
@@ -1031,6 +1031,14 @@ public static byte[] getBytesWithEncoding(String string) {
         return string.getBytes(StandardCharsets.UTF_8);
     }
 
+    /**
+     * @param value string to process
+     * @return string without any whitespaces formatted to lowercase.
+     */
+    public static String toLowerWithoutWhitespaces(String value) {
```
Comment (nvoxland) [inline liquibase-core/src/main/java/liquibase/util/StringUtil.java:1038]: This should be updated to return null if value is null instead of throwing a nullpointerexception

**Review by nvoxland — APPROVED** [review]
Added one comment on something good to fix, but still marking it as approved so I don't have to be a bottleneck in re-approving it

File: liquibase-core/src/main/java/liquibase/util/StringUtil.java (hunk 1 above)
Comment (dyadyushko) [reply liquibase-core/src/main/java/liquibase/util/StringUtil.java:1040]: Nice catch, going to fix!

File: liquibase-core/src/main/java/liquibase/util/StringUtil.java (hunk 3 above)
Comment (dyadyushko) [reply liquibase-core/src/main/java/liquibase/util/StringUtil.java:1038]: Definitely, gonna do that
